doc/source/cd/methods/onlinecvmdrift.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<p>Lets mention at top that it's for continuous data.</p><p>"Therefore, it is recommended that users keep&nbsp;M reasonably small" - is a little ambiguous as to what we can handle. I think something more along the lines of "we recommend using fewer than 10,000 instances as reference".</p><p>Maybe note ERT still lower bounded in non-independent multivariate case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>